### PR TITLE
Update master for 3.12 branching

### PR DESCRIPTION
**Pull request by ColeHiggins2**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: web/releases/3.12.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+=== Quickstart
+
+* link:/{FOREMAN_VER}/Quickstart/index-foreman-el.html[Foreman on RHEL/CentOS quickstart guide]
+* link:/{FOREMAN_VER}/Quickstart/index-foreman-deb.html[Foreman on Debian/Ubuntu quickstart guide]
+* link:/{FOREMAN_VER}/Quickstart/index-katello.html[Katello on RHEL/CentOS quickstart guide]
+
+=== Installation
+
+* link:/{FOREMAN_VER}/Installing_Server/index-foreman-el.html[Installing Foreman on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server/index-foreman-deb.html[Installing Foreman on Debian/Ubuntu]
```
Comment (ekohl) [inline web/releases/3.12.adoc:17]: These aren't ready yet
```suggestion
```

File: web/releases/3.12.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* link:/{FOREMAN_VER}/Installing_Server/index-katello.html[Installing Katello on RHEL/CentOS]
+
+* link:/{FOREMAN_VER}/Installing_Proxy/index-foreman-el.html[Installing Smart Proxy on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Proxy/index-foreman-deb.html[Installing Smart Proxy on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Installing_Proxy/index-katello.html[Installing Smart Proxy with content on RHEL/CentOS]
+
+=== Upgrading
+
+* link:/{FOREMAN_VER}/Release_Notes/index-foreman-el.html[Release notes for Foreman on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Release_Notes/index-foreman-deb.html[Release notes for Foreman on Debian/Ubuntu]
```
Comment (ekohl) [inline web/releases/3.12.adoc:27]: This guide isn't ready yet
```suggestion
```

File: web/releases/3.12.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* link:/{FOREMAN_VER}/Quickstart/index-katello.html[Katello on RHEL/CentOS quickstart guide]
+
+=== Installation
+
+* link:/{FOREMAN_VER}/Installing_Server/index-foreman-el.html[Installing Foreman on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Server/index-foreman-deb.html[Installing Foreman on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Installing_Server/index-katello.html[Installing Katello on RHEL/CentOS]
+
+* link:/{FOREMAN_VER}/Installing_Proxy/index-foreman-el.html[Installing Smart Proxy on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Proxy/index-foreman-deb.html[Installing Smart Proxy on Debian/Ubuntu]
```
Comment (ekohl) [inline web/releases/3.12.adoc:21]: These guides aren't ready yet
```suggestion
```

File: web/releases/3.12.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* link:/{FOREMAN_VER}/Installing_Proxy/index-katello.html[Installing Smart Proxy with content on RHEL/CentOS]
+
+=== Upgrading
+
+* link:/{FOREMAN_VER}/Release_Notes/index-foreman-el.html[Release notes for Foreman on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Release_Notes/index-foreman-deb.html[Release notes for Foreman on Debian/Ubuntu]
+* link:/{FOREMAN_VER}/Release_Notes/index-katello.html[Release notes for Katello on RHEL/CentOS]
+
+// Upgrading guides are not ready for non-Katello
+//* link:/{FOREMAN_VER}/Upgrading_Project/index-foreman-el.html[Upgrading Foreman on RHEL/CentOS]
```
Comment (ekohl) [inline web/releases/3.12.adoc:31]: We actually declared this guide as ready
```suggestion
* link:/{FOREMAN_VER}/Upgrading_Project/index-foreman-el.html[Upgrading Foreman on RHEL/CentOS]
```

File: web/releases/3.12.json [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+          "path": "Quickstart"
+        },
+        {
+          "title": "Installing Foreman server",
+          "path": "Installing_Server"
+        },
+        {
+          "title": "Installing Smart Proxy",
+          "path": "Installing_Proxy"
+        },
```
Comment (ekohl) [inline web/releases/3.12.json:24]: These aren't ready yet
```suggestion
```

File: web/releases/3.12.json [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+          "path": "Deploying_Project_on_AWS"
+        },
+        {
+          "title": "Provisioning hosts",
+          "path": "Provisioning_Hosts"
+        },
+        {
+          "title": "Managing hosts",
+          "path": "Managing_Hosts"
+        },
```
Comment (ekohl) [inline web/releases/3.12.json:40]: Also not ready
```suggestion
```

File: web/releases/3.12.json [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+          "path": "Managing_Security_Compliance"
+        },
+        {
+          "title": "Administering Foreman",
+          "path": "Administering_Project"
+        },
+        {
+          "title": "Application centric deployment",
+          "path": "Deploying_Hosts_AppCentric"
+        },
```
Comment (ekohl) [inline web/releases/3.12.json:64]:
```suggestion
```

File: web/releases/3.12.json [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+          "path": "Installing_Server"
+        },
+        {
+          "title": "Deploying Foreman on AWS",
+          "path": "Deploying_Project_on_AWS"
+        },
+        {
+          "title": "Provisioning hosts",
+          "path": "Provisioning_Hosts"
+        },
```
Comment (ekohl) [inline web/releases/3.12.json:94]:
```suggestion
```

File: web/releases/3.12.json [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+          "path": "Managing_Configurations_Ansible"
+        },
+        {
+          "title": "Configuring hosts by using Puppet",
+          "path": "Managing_Configurations_Puppet"
+        },
+        {
+          "title": "Administering Foreman",
+          "path": "Administering_Project"
+        },
```
Comment (ekohl) [inline web/releases/3.12.json:106]:
```suggestion
```

**Review by ekohl — CHANGES_REQUESTED** [review]

**Review by ekohl — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -11,6 +11,7 @@
 
 Please cherry-pick my commits into:
 
+* [ ] Foreman 3.12/Katello 4.14 (Satellite 6.16)
```
Comment (maximiliankolb) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: @ekohl Is this correct? :eyes:

Comment (ekohl) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: Yes

**Pull request merged**